samples/event-sourced-customer-registry/src/test/java/customer/domain/CustomerEventSerializationTest.java

```diff
@@ -17,7 +15,7 @@ class CustomerEventSerializationTest {
   @Test
   public void shouldDeserializeWithMandatoryField() {
     //given
-    Any serialized = JsonSupport.encodeJson(new CustomerEvent.NameChanged("andre"));
+    ByteString serialized = JsonSupport.encodeToAkkaByteString(new CustomerEvent.NameChanged("andre"));
```

Comment:
This doesn't work, because I think we are loosing some migration. Somewhat strange for this test case, but understand that it will be a problem for that other cases.

Not sure what we shall do? Using pb Any in our docs doesn't make sense. BytesPayload could be a replacement but that is not intended to be user facing. What do you think @aludwiko ?